sources.go

```diff
@@ -45,6 +46,7 @@ func IfExists(filename string) *Pipe {
 // starting pipelines. If there is an error opening the file, the pipe's error
 // status will be set.
 func File(name string) *Pipe {
+	name = strings.TrimRight(name, "\n")
```

Comment:
Same here; the `name` doesn't come from a pipe, so we can leave it as is.

Reply:
It strikes me that it would make more sense for the kind of use case you described to trim trailing newlines in _filter_ methods such as Concat, wouldn't it? But in fact that's not necessary; consider this program:

```go
	script.Args().Concat().Stdout()
```

This works exactly as you'd expect.